packages/fiori-freestyle/src/index.ts

```diff
@@ -102,7 +102,12 @@ async function generate<T>(basePath: string, data: FreestyleApp<T>, fs?: Editor)
 function copyTemplates(tmplPath: string, basePath: string, ffApp: FreestyleApp<unknown>, fs: Editor) {
     // Remove odata versions specific path while copying template files
     const replaceVer = `\\${sep}v${Object.values(OdataVersion).join(`|\\${sep}v`)}`;
-    const removeVersionTmplPath = (path: string): string => path.replace(new RegExp(replaceVer), '');
+    const destPathManipulations = (path: string): string => {
+        return path
+            .replace(new RegExp(replaceVer), '') // remove version template path
```

Comment:
Allowing to provide a custom view/controller name sounds good. I would like to see the defaulting in one place i.e. https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application/src/data/defaults.ts#L51 instead of in all `ejs` templates and here.
Btw, would you add, both view and controller name as parameter or wouldn't it be better style to just ask for the view name and make sure the controller name matches? I am aware that it is technically possible to have different names but I am wondering if we should enforce best practices.